e2e/testcases/remediator_test.go

```diff
@@ -45,17 +45,19 @@ func TestSurfaceFightError(t *testing.T) {
 	nt.Must(nt.WatchForAllSyncs())
 
 	// Make the # of updates exceed the fightThreshold defined in pkg/syncer/reconcile/fight_detector.go
+	doneCh := make(chan struct{})
 	go func() {
-		for i := 0; i <= 5; i++ {
+		defer close(doneCh)
+		for i := 0; i < 20; i++ {
 			nt.MustMergePatch(ns, `{"metadata": {"annotations": {"foo": "baz"}}}`)
-			nt.MustMergePatch(rb, `{"metadata": {"annotations": {"foo": "baz"}}}`)
-			time.Sleep(time.Second)
+			time.Sleep(500 * time.Millisecond)
 		}
 	}()
 
 	nt.T.Log("The RootSync reports a fight error")
 	nt.Must(nt.Watcher.WatchForRootSyncSyncError(configsync.RootSyncName, status.FightErrorCode,
 		"This may indicate Config Sync is fighting with another controller over the object.", nil))
+	<-doneCh // Wait for fighting goroutine to exit
 
 	rootSyncNN := nomostest.RootSyncNN(configsync.RootSyncName)
 	rootSyncLabels, err := nomostest.MetricLabelsForRootSync(nt, rootSyncNN)
```

Comment:
Need to update the metric criteria

Reply:
Which metric criteria needs to change? The test is passing when I ran it locally